public/index.html

```diff
@@ -45,11 +45,11 @@ <h2 class="Polaris-Heading">New Survey</h2>
                         <div class="Polaris-FormLayout__Item">
                           <div class="">
                             <div class="Polaris-Labelled__LabelWrapper">
-                              <div class="Polaris-Label"><label id="PolarisTextField20Label" for="PolarisTextField20" class="Polaris-Label__Text">Title</label></div>
+                              <div class="Polaris-Label"><label id="PolarisTextField20Label" for="PolarisTextField20" class="Polaris-Label__Text">Name</label></div>
                             </div>
                             <div class="Polaris-Connected">
                               <div class="Polaris-Connected__Item Polaris-Connected__Item--primary">
-                                <div class="Polaris-TextField"><input id="PolarisTextField20" class="Polaris-TextField__Input" type="text" name="title">
+                                <div class="Polaris-TextField"><input id="PolarisTextField20" class="Polaris-TextField__Input" type="text" name="Name">
```

Comment:
```suggestion
                                <div class="Polaris-TextField"><input id="PolarisTextField20" class="Polaris-TextField__Input" type="text" name="name">
```

change to `name` to match the `@RequestParam('name')`